crates/starknet/src/starknet/mod.rs

```diff
@@ -738,4 +743,14 @@ mod tests {
 
         assert_eq!(num_one_transaction.unwrap(), 1);
     }
+
+    #[test]
+    fn returns_chain_id() {
```

Comment:
You can also add an e2e test. You would need to modify the `spawn` method of `BackgroundDevnet` to include the chain id CLI parameter

Reply:
Actually that would require quite a lot of changes, so you might as well skip this suggestion.

Reply:
yes I was checking that but this PR would change to BackgroundDevnet CLI parameter PR